apiserver/params/params.go

```diff
@@ -485,6 +489,27 @@ type ApplicationGetConfigResults struct {
 	Results []ConfigResult
 }
 
+// LXDProfileUpgradeMessages holds the parameters for an application
+// lxd profile machines
+type LXDProfileUpgradeMessages struct {
+	ApplicationTag string `json:"application"`
```

Comment:
tags are usually passed as Entity types.

Reply:
I'd agree, but I also need to pass the watcher id as well. So should we just rename this or remove the idea of the tag, which is a shame, because validating on the tag is actually quite useful.

Reply:
let's leave the tag, it should just be of type Entity.  s/ApplicationTag string/ ApplicationTag Entity/